starters/nuxt-2/components/ProductCard.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div v-if="product" class="product-card">
+    <nuxt-link
+      :to="`/products/${product.content.handle}`"
+      class="product-card__media"
+    >
+      <nuxt-img
+        v-if="product.content.featuredMedia"
+        :src="product.content.featuredMedia.src"
+        class="product-card__image"
+      />
+      <div v-else>No Image</div>
+    </nuxt-link>
+    <div class="product-card__main">
+      <h3 v-if="product.content.title" class="product-card__title">
```

Comment:
Based on how we're using this in the starter, this should probably be an `h2`. But it's one of those places where components butt up against html since there may be instances where this actually needs to be an `h3`.